Update AnimalGenomicSIG meeting schedule (#2771)
nf-core/website: sites/main-site/src/content/events/2024/SIG_animalgenomics_Nov.md

---
title: '#animal-genomics: November Meeting'
subtitle: Harris Lewin (Arizona State University, USA)

type: talk
startDate: '2024-11-20'
startTime: '17:00+02:00'
endDate: '2024-11-20'
endTime: '18:00+02:00'
locations:
  - name: Online
    links:
      - https://rediris.zoom.us/j/96065549412?pwd=L2txanpQMEFTOWVLNmtIZyt6M3NnUT09
---

November 2024 talk of the monthly meeting of the _#animal-genomics_ special interest group.

> In this meeting **Harris Lewin** (Arizona State University, USA) will speak about the [Earth BioGenome Project](https://www.earthbiogenome.org/).

Reviewer: Just to avoid the same structure in the both sentences.
```suggestion
**Harris Lewin** (Arizona State University, USA) will discuss the [Earth BioGenome Project](https://www.earthbiogenome.org/) during this meeting.
```
